src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -4,8 +4,49 @@
 
 """
 
+from typing import Tuple
+from ...molecules import BuildingBlock
 from .vertices import HostVertex, GuestVertex
 from ..topology_graph import TopologyGraph, NullOptimizer
+from dataclasses import dataclass
+import warnings
+
+
+@dataclass(frozen=True)
+class Guest:
+    """
+    Holds the data defining the placement of a guest molecule.
+
+    Attributes
+    -------------
```

Comment:
Is this the correct number of underscores? Not sure if it's just the GitHub rendering being weird.